tests/integration/test_data/pip_multiple/bom.json

```diff
@@ -40,9 +40,153 @@
           "value": "second_pkg/requirements.txt"
         }
       ],
-      "purl": "pkg:pypi/appr?vcs_url=git%2Bhttps://github.com/quay/appr.git%4058c88e4952e95935c0dd72d4a24b0c44f2249f5b",
+      "purl": "pkg:pypi/appr?vcs_url=git%2Bhttps://github.com/quay/appr.git%4037ff9a487a54ad41b59855ecd76ee092fe206a84",
       "type": "library"
     },
+    {
+      "name": "async-timeout",
+      "properties": [
+        {
+          "name": "cachi2:found_by",
+          "value": "cachi2"
+        },
+        {
+          "name": "cachi2:missing_hash:in_file",
+          "value": "second_pkg/requirements.txt"
+        }
+      ],
```

Comment:
Do we want a case where a dependency is missing hashes in more than one file?

Reply:
Added.